partcad/src/partcad/project.py

```
@@ -803,8 +803,13 @@ def get_object(
                     pc_logging.error("Failed to instantiate a non-parametrized object %s" % object_name)
                 return objects[object_name]
 
-            # This object has params (part_name != result_name)
-            if not base_object_name in objects:
+            # This object has params (part_name != result_name). Only the base
+            # object's *config* is needed to derive the parametrized variant
+            # (see 'object_configs[base_object_name]' below), so check the
+            # enumerable configs rather than the instantiated 'objects' dict -
+            # a plugin-backed package enumerates lazily and may not have
+            # instantiated the base yet.
+            if not base_object_name in object_configs:
```

Ruff (0.15.21)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)